src/main/java/com/todolist/app/servlet/SeeAllTasksServlet.java

```diff
@@ -0,0 +1,48 @@
+package com.todolist.app.servlet;
+
+import com.todolist.app.model.Item;
+import com.todolist.app.model.Status;
+import com.todolist.app.service.ToDoStoreSingleton;
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+
+@WebServlet(name = "SeeAllTasksServlet", value = "/see-all-tasks")
+public class SeeAllTasksServlet extends HttpServlet {
+    static ToDoStoreSingleton singleton = ToDoStoreSingleton.getInstance();
+
+    static {
+        singleton.getExample();
```

Comment:
Could you move this method to the constructor of the singleton because it's the responsibility of singleton to fill in the store?